servers/admin_ui/landing.py

```diff
@@ -0,0 +1,23 @@
+import streamlit as st
+
+from servers.admin_ui.common import print_header
+
+# configure streamlit
+st.set_page_config(layout="wide")
+print_header(title="Agent Guard", sub_title="Securing Agentic AI")
+
+st.write("\n\n")
+st.subheader("Welcome to Agent Guard!")
+st.write("""
+Agent Guard is your AI-driven security solution, offering the following capabilities:
+- **Secret Provider**: Acts as a proxy to the following secret providers:
+    - AWS Secrets Manager
+    - CyberArk Conjur secret provider
+    - Local file secret provider (for testing purposes and non sensitive data only) 
```

Comment:
Where is that stored? In the filesystem of the server? Is that secure? What if someone adds malicious content?